File: SDKs/github.com/ghes-2.22/1.1.4/go/pkg/models/shared/orgenum.go
package shared

type OrgEnum string

const (
	OrgEnumAssigned   OrgEnum = "assigned"
	OrgEnumCreated    OrgEnum = "created"
	OrgEnumMentioned  OrgEnum = "mentioned"
	OrgEnumSubscribed OrgEnum = "subscribed"
	OrgEnumRepos      OrgEnum = "repos"
	OrgEnumAll        OrgEnum = "all"
)

File: SDKs/github.com/ghes-2.22/1.1.4/java/lib/src/main/java/openapisdk/models/shared/OrgEnum.java
package openapisdk.models.shared;


public enum OrgEnum {
    ASSIGNED("assigned"),
    CREATED("created"),
    MENTIONED("mentioned"),
    SUBSCRIBED("subscribed"),
    REPOS("repos"),
    ALL("all");

    public final String value;

    private OrgEnum(String value) {
        this.value = value;
    }
}
